docs/manage/ml/_index.md

```diff
@@ -5,18 +5,25 @@ weight: 40
 type: "docs"
 tags: ["data management", "data", "services"]
 no_list: true
-description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots"
+description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots."
 # SME: Aaron Casas
 ---
 
-Machine learning is an approach to programming where you provide an algorithm with a large data set, and train it to discover patterns in that data on its own.
-The more data you provide to it, the better it becomes at identifying patterns across that data set, and the more accurate it becomes at guessing characteristics of any new data provided.
+Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
 
-For example, say you have a collection of pictures of your cat.
-As you train your algorithm on just a few pictures, it might not be able to determine that a picture of your cat's face and a picture of your cat sleeping are the same cat.
-As you train the algorithm on more and more pictures, it might be able to recognize your cat from any angle, or to differentiate between multiple different cats.
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
```

Comment:
I think those would still be tensorflow lite models - just user provided ones, so I think more correct to say:

```suggestion
Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
```